### PR TITLE
Added un analyzed topic

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Added message topic
related - https://github.com/thoth-station/graph-refresh-job/issues/433

### CODE REVIEW

File: thoth/messaging/unanalyzed_package.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+"""This is Thoth Messaging module for UnanalyzedPackageMessage."""
+
+import logging
+
+from .message_base import MessageBase, BaseMessageContents
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class UnanalyzedPackageMessage(MessageBase):
```
Comment (pacospace) [inline thoth/messaging/unanalyzed_package.py:27]: should we called `SIUnanalyzedPackageMessage`?

Comment (saisankargochhayat) [reply thoth/messaging/unanalyzed_package.py:27]: Makes more sense! Thanks

File: thoth/messaging/unanalyzed_package.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    """Class used by Producer events on Kafka topic on packages not analyzed by SI."""
+
+    topic_name = "thoth.investigator.unanalyzed-package"
```
Comment (saisankargochhayat) [inline thoth/messaging/unanalyzed_package.py:30]: "thoth.investigator.si-unanalyzed-package"?? @pacospace

Comment (pacospace) [reply thoth/messaging/unanalyzed_package.py:30]: sounds great! Thanks!!

File: thoth/messaging/__init__.py [hunk 3]
```diff
@@ -27,6 +27,7 @@
 from .solved_package import SolvedPackageMessage
 from .unresolved_package import UnresolvedPackageMessage
 from .unrevsolved_package import UnrevsolvedPackageMessage
+from .unanalyzed_package import SIUnanalyzedPackageMessage
```
Comment (pacospace) [inline thoth/messaging/__init__.py:30]: si_unanalyzed_package?

**Review by pacospace — APPROVED** [review]
Thanks!!

**Pull request merged**